### ISSUE TITLE
Baptism Badge classname has spaces in it "badge - baptism"

### ISSUE THREAD

**Issue by nairdo** [SparkDevNetwork/Rock#1591]
...and therefore it incorrectly appears white.  I've confirmed this on CentralAZ's system which is somewhere around v5 (or slightly after v5).  I also see this issue on our Spark server.

```
{% assign baptismDate = Person | Attribute:'BaptismDate' %}

{% if baptismDate != '' -%}
    <div class="badge badge - baptism" data-toggle="tooltip" data-original-title="{{ Person.NickName }} was baptized on {{ baptismDate }}.">
<i class="badge-icon fa fa-tint"></i>
    </div>
{% else -%}
    <div class="badge badge-baptism" data-toggle="tooltip" data-original-title="No baptism date entered for {{ Person.NickName }}.">
        <i class="badge-icon badge-disabled fa fa-tint"></i>
    </div>
{% endif -%}
```

**Comment by edmistj** [comment]
I think I fixed the migration that causes this.

**Comment by nairdo** [comment]
Just tried to reproduce it on a new install using the new Rock v5 alpha and was not able to, so I suspect you got it.  I found your fix over at dd9c6553e7422e7a46623bf1971cb9e149351f8f.
